patches/chrome-test-data-webui-settings-a11y_page_test.ts.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/test/data/webui/settings/a11y_page_test.ts b/chrome/test/data/webui/settings/a11y_page_test.ts
+index f549afbc44ed93e2b907a8a705534d3ce4cad189..40586a904539e1cfe6a80fd4370a4922b91b9f4b 100644
+--- a/chrome/test/data/webui/settings/a11y_page_test.ts
++++ b/chrome/test/data/webui/settings/a11y_page_test.ts
+@@ -62,7 +62,7 @@ suite('A11yPage', () => {
+     });
+   });
+ 
+-  setup(async function() {
```

Comment:
this doesn't seem right, why are we modifying an upstream test page?

Reply:
oic it's wip

Reply:
I am actually not sure why we are seeing an error here while upstream doesn't.

Reply:
why are we even running lint on upstream sources?

Reply:
so, this file is included into `"//chrome/test/data/webui/settings:build_ts"`, which is a dependency of `//third_party/polymer/v3_0:library` , which is in ts_deps of `//brave/ui/webui/resources:build` - a `build_webui` and the upstream change makes `build_webui` run eslint.

Reply:
are there options to ignore certain directories/files?

Reply:
Maybe we can supply our own `tsconfig` file in `//brave/ui/webui/resources:build` and in there add `"exclude"` section? cc: @fallaciousreasoning would that be possible?

Reply:
Ok, so this doesn't work:
 ```AssertionError: Invalid |exclude| option detected in brave\ui\webui\resources\tsconfig_base.json.Only |extends| and |compilerOptions| may be specified.```

Reply:
Created a follow up issue https://github.com/brave/brave-browser/issues/44711